inference-engine/ie_bridges/java/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 Intel Corporation
+
+cmake_minimum_required(VERSION 3.0.0)
+
+find_package(Java REQUIRED)
+include(UseJava)
+
+set(JAVA_AWT_INCLUDE_PATH NotNeeded)
+
+project(inference_engine_java_api)
+
+if(DEFINED IE_MAIN_SOURCE_DIR)
+    set(InferenceEngine_LIBRARIES inference_engine)
+else()
+    find_package(InferenceEngineDeveloperPackage REQUIRED)
+    set(InferenceEngine_LIBRARIES IE::inference_engine)
+endif()
```

Comment:
I think we don't build against `InferenceEngineDeveloperPackage`, Let's assume that `IE_MAIN_SOURCE_DIR` is always defined